app/views/guide_typography.html

```diff
@@ -214,6 +215,23 @@ <h3 class="heading-medium" id="typography-hidden-text">Hidden text (progressive
   </div>
 
 
+  <h3 class="heading-medium" id="typography-class">Use styling automatically</h3>
+  <div class="text">
+    <p>You can use the <code class="code">govuk-prose</code> class around a body of prose. Then certain elements inside will be styled automatically without the need to add specific classes to each of them.</p>
+    <p>That will apply to most typographic elements: headings, lists, links (without govuk_template), code and strong. Tables, paragraphs, links (with govuk_template) and horizontal rules are already always styled without that class.</p>
```

Comment:
Should we also warn that you should avoid nesting components within .govuk-prose?